cpp/daal/src/algorithms/dtrees/forest/classification/oneapi/df_classification_train_hist_oneapi_impl.i

```diff
@@ -392,7 +417,7 @@ services::Status ClassificationTrainBatchKernelOneAPI<algorithmFPType, hist>::co
         args.set(10, partialHistograms, AccessModeIds::write);
         args.set(11, static_cast<int32_t>(nSelectedFeatures));
 
-        size_t localSize = _preferableGroupSize;
+        size_t localSize = _preferableLocalSizeForPartHistKernel;
```

Comment:
Why don't you use this variable directly?

Reply:
it is suitable to have it separated in case if it is calculated in place via other vars.